test/e2e/v1beta1/scripts/gh-actions/build-load.sh

```diff
@@ -162,6 +162,10 @@ for name in "${TRIAL_IMAGE_ARRAY[@]}"; do
   run "$name" "examples/$VERSION/trial-images/$name/Dockerfile"
 done
 
+# Testing image for tune function
+echo -e "\nPulling and building testing image for tune function..."
+_build_containers "suggestion-hyperopt" "$CMD_PREFIX/suggestion/hyperopt/$VERSION/Dockerfile"
+
```

Comment:
According to the @tenzen-y comment here: https://github.com/kubeflow/katib/pull/2399#discussion_r1700574562, do we want to refactor this script to always build all suggestion + trial images ?

Reply:
I guess we may not want to build all suggestion + trials images, since it will definitely raise the time for CI/CD test. We only need to build necessary images, such as `suggesion-hyperopt` for random search algorithm. 

WDYT👀 @andreyvelich @tenzen-y

Reply:
Indeed, you are right.
In that case, can you check if this e2e has `tune-api` parameter when you build image for `suggestion-hyperopt` ?

Reply:
SGTM.